src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -43,6 +36,13 @@
       </div>
     </v-fade-transition>
 
+    <!-- Court order notification -->
+    <v-card v-if="hasCourtOrders" class="my-6 pa-6" elevation="0">
+      <v-icon>mdi-gavel</v-icon>
+      <span class="ml-1">Court order(s) have been filed on this company. Review the filing history for impacts
+        to business information.</span>
+    </v-card>
+
```

Comment:
I moved this down to be consistent with other components where we first have dialogs, then the spinner, then the page components.